workers/tests/integrationtests/test_api_indicator.py

```diff
@@ -21,9 +21,9 @@ class TestApiIndicator(unittest.TestCase):
     def setUp(self):
         self.client = TestClient(app)
         self.dataset = "regions"
-        self.feature_id = 31
+        self.feature_id = 1
         self.bpolys = asyncio.run(
-            db_client.get_bpolys_from_db(self.dataset, self.feature_id)
+            db_client.get_bpolys_from_db(self.dataset, self.feature_id, "ogc_fid")
```

Comment:
Why is `"ogc_fid"` hardcoded here and not in a class variable like `self.dataset`? The same question applies to all the other tests.

Reply:
I changed it to be consistent with the other code